diff --git a/dist/chunk-QDCFQH3Q.mjs b/dist/chunk-QDCFQH3Q.mjs
index 225d94a28e9e0a1f799907b7e07fde55d79c4aa6..205b60f23f4cb68289b3abbda3745f3347b43112 100644
--- a/dist/chunk-QDCFQH3Q.mjs
+++ b/dist/chunk-QDCFQH3Q.mjs
@@ -39,7 +39,6 @@ function usePaginationItem(props) {
   const getItemProps = (props2 = {}) => {
     return {
       ref: domRef,
-      role: "button",
       tabIndex: isDisabled ? -1 : 0,
       "aria-label": ariaLabel,
       "aria-current": dataAttr(isActive),
