diff --git a/dist/chunk-B7FBUABT.mjs b/dist/chunk-B7FBUABT.mjs
index 2b8c21209fee3dd4d49ce7b2293ada6f04e2f422..a315b98cffae1b8dfb07ff109c00149ef423f295 100644
--- a/dist/chunk-B7FBUABT.mjs
+++ b/dist/chunk-B7FBUABT.mjs
@@ -42,7 +42,7 @@ function useHiddenSelect(props, state, triggerRef) {
     },
     inputProps: {
       type: "text",
-      tabIndex: modality == null || state.isFocused || state.isOpen ? -1 : 0,
+      tabIndex: -1,
       autoComplete,
       value: (_a = [...state.selectedKeys].join(",")) != null ? _a : "",
       required: isRequired,
